sqlite.go

```diff
@@ -556,3 +562,13 @@ func (err Error) Error() string {
 	}
 	return b.String()
 }
+
+// WithPersist makes a ctx instruct the sqlite driver to persist a prepared query.
+//
+// This should be used with recurring queries to avoid constant parsing and
+// planning of the query by SQLite.
+func WithPersist(ctx context.Context) context.Context {
+	return context.WithValue(ctx, persistQuery(0), persistQuery(0))
+}
+
+type persistQuery int
```

Comment:
use an empty struct to avoid boxing into an interface{}

and document that it's a context value

Reply:
done.